ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -744,9 +755,21 @@ export default class ConfirmTransactionBase extends Component {
     if (toAddress) {
       tryReverseResolveAddress(toAddress);
     }
+
+    getGasFeeEstimatesAndStartPolling().then((pollingToken) => {
```

Comment:
add some docs.

Reply:
- [ ] Add documentation

This is handling the case describe in this comment on a separate PR: https://github.com/MetaMask/metamask-extension/pull/11511/files#r670686360